drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -93,15 +93,50 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
           }
         end
 
-      when 'REFERRAL_POSTING_STATUSES'
-        HmisExternalApis::AcHmis::ReferralPosting::VALID_LOCAL_STATUSES.map do |status|
+      when 'ASSIGNED_REFERRAL_POSTING_STATUSES'
+        HmisExternalApis::AcHmis::ReferralPosting::ASSIGNED_STATUSES.map do |status|
           {
             code: status,
             label: status.gsub(/_status\z/, '').humanize.titleize,
           }
         end
+      when 'DENIED_PENDING_REFERRAL_POSTING_STATUSES'
+        label_map = {
+          'assigned_status' => 'Send Back',
+          'denied_status' => 'Approve Denial',
+          'denied_pending_status' => 'Denied Pending',
```

Comment:
`denied_pending_status` shouldn't be an option for the denial screen. It's just "send back" or "approve denial"

Reply:
Good to know. I included it since it's the initial value of the select form control. [I adjusted the form to set it to null](https://github.com/greenriver/hmis-frontend/pull/321/files#diff-057a6f90538096932ab45bae799958f34772b6c6436e6839012cdca382f45fdfR29)